dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -145,6 +145,19 @@ def show
     @level = select_level
     return if redirect_under_13_without_tos_teacher(@level)
 
+    # TODO: If this adds too much to the load time in prod, move it to an API.
+    if current_user&.teacher?
+      @responses = []
+      # We use this for the level summary entry point, so on contained levels
+      # what we actually care about are responses to the contained level.
+      level = @level.contained_levels.any? ? @level.contained_levels.first : @level
+
+      # TODO: Change/remove this check as we add support for more level types.
+      if level.class.to_s == 'FreeResponse'
```

Comment:
i didn't see a const for this anywhere obvious - if anyone knows a cleaner way to check this i'd be happy to hear it

Reply:
Could you do something like `if level.is_a?(FreeResponse`?